### DISCUSSION TITLE
Farsi(Persian) Doc Translation

### DISCUSSION THREAD

**Discussion by alirezahamid** [vuejs/pinia#2508] [Ideas]
Hello @posva,

I'm Alireza, a Pinia user, proposing to translate its documentation into Farsi. This effort aims to make Pinia more accessible to Farsi-speaking developers, enhancing its global reach.

As a fluent speaker of both English and Farsi, with technical translation experience, I am confident in providing an accurate and useful translation. I understand the importance of maintaining Pinia's documentation standards and am ready to collaborate closely with your team.

Please advise on how I can start this project. I'm eager to contribute to Pinia’s growth and look forward to any guidance you can offer.

Thank you for considering my proposal.

Best regards.

**Comment by mostafa-nematpour** [comment]
Dear @posva 

We're seeking permission to translate Pinia documentation into Persian. This translation will greatly benefit Persian-speaking developers, enabling a more comprehensive utilization of Pinia's features within our community.

Your approval for this translation would be immensely appreciated.

Thank you.

**Comment by posva** [comment]
Of course! Feel free to follow any of the other translation PRs and repositories.

**Reply by alirezahamid** [reply]
Thanks for reply. Is there any policy which we can have the Farsi version as https://pinia.vuejs.org/fa/ ?

**Reply by posva** [reply]
To simplify the maintaince at the moment, it will have to be on an external domain like all other translations. Only English and Chinese are kept here
